src/mission.cpp

```diff
@@ -2496,6 +2496,32 @@ Before we get into a major fight just make sure we have the gear we need, boss."
         }
         break;
 
+    case MISSION_FIND_ZSG:
+        if( state == "TALK_MISSION_DESCRIBE" ) {
+            return _("I've can an idea about how to improve our chances of surviving");
```

Comment:
You push to your branch (mlangsdorf:fix_mission_78) and this PR automatically gets updated.
Pro tip: use a GUI client for git stuff, such as SourceTree or GitHub for Windows.